src/test/java/com/work/management/service/employee/impl/EmployeeServiceImplTest.java

```diff
@@ -48,10 +55,42 @@ public void updateEmployeeEntity_forExistingEmployee_returnsUpdatedEmployee() {
     assertThat(actualEmployeeDto).isEqualTo(employeeDto);
   }
 
-  private static EmployeeDto getEmployeeDto(Employee employee) {
-    EmployeeDto employeeDto = new EmployeeDto();
-    BeanUtils.copyProperties(employee, employeeDto);
-    return employeeDto;
+  @Test
+  public void bulkUpdate_forExistingEmployees_returnUpdatedEmployeesList() {
+    ImmutableList<Employee> employeeList = getEmployeeList();
+    employeeList.forEach(employee -> employeeService.save(getEmployeeDto(employee)));
+
+    ImmutableList<Employee> actualBulkUpdatedEmployeesInfo = employeeService
+        .bulkUpdate(BulkEmployeeDto.builder()
+            .employeeIds(ImmutableList.of(1, 2, 8))
+            .acceptedFieldsMap(ImmutableMap.<AcceptedFields, String>builder()
+                .put(AcceptedFields.MANAGER_ID, "10")
+                .put(AcceptedFields.TEAM_ID, "c7f6fbab-22fb-41bc-9300-0cc27c0de5c5").build())
+            .build());
+    ImmutableList<Employee> updatedEmployeesInfo = employeeList.stream()
+        .map(employee -> employeeService.getEmployeeByUserName(employee.getUserName()))
+        .map(EmployeeServiceImplTest::getEmployee)
+        .collect(toImmutableList());
```

Comment:
create method readEmployeesByUsername and use that directly in the assert statement, instead of creating a variable.